journalpump/journalpump.py

```diff
@@ -249,15 +251,35 @@ def __init__(self, config, msg_buffer, stats):
         self.index_days_max = self.config.get("elasticsearch_index_days_max", 3)
         self.index_name = self.config.get("elasticsearch_index_prefix", "journalpump")
         self.es = Elasticsearch([self.elasticsearch_url], timeout=self.request_timeout)
+        self.indices = {key for key in self.es.indices.get_aliases().keys()}
```

Comment:
what does get_aliases() return.  a dict?  just `set(self.es.indices.get_aliases())` would be enough here - no need to explicitly iterate over keys.